arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -4,11 +4,55 @@
     "version": "2016-06-27-preview",
     "title": "ContainerRegistry"
   },
-  "host": "https://management.azure.com",
+  "host": "management.azure.com",
   "schemes": [
-    "http"
+    "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/moveResources": {

```

Comment:
The command for resource move is "az resource move ..." and we shouldn't expose this API in "az acr" group, but we should expose check name availability API here.